packages/common/package.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "@axa-fr/common-design-system",
```

les glyphicons ne sont utilisés que pour slash.
Après avoir discuter avec @arnaudforaison sur mon dev qui converge Apollo et L&F, on avait décidé de nommer les dossiers common par DS pour plus de clareté.
```suggestion
  "name": "@axa-fr/common-slash-design-system",
```

---

@samuel-gomez-axa est dans ce cas il y a du nettoyage à prévoir coté LF car  c'est utiliser ici https://github.com/AxaFrance/design-system/blob/main/client/look-and-feel/css/src/common/icons.scss